crates/daphne-worker/clippy.toml

```diff
@@ -2,5 +2,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 disallowed-methods = [
-    { path = "std::time::Instant::now", reason = "not implemented in wasm" },
+    { path = "std::time::Instant::now", reason = "not implemented in wasm. Use worker::Date::now()" },
+    { path = "std::time::SystemTime::now", reason = "not implemented in wasm. Use worker::Date::now()" },
```

Comment:
What does this change do, for my own edification?

Reply:
It forbids the use of the standard library's time api which simply crash in wasm. This way we get a clippy error